.changeset/public-files-throw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/astro': minor
+---
+
+Introducing `<experimental_PricingTable/>`
```

Comment:
```suggestion
Add `<Experimental_PricingTable />` component to the SDK. Learn more about it in the [Clerk Billing guide](https://clerk.com/docs/billing/overview)
```

afaik this will be the final URL of the guide